ensime-startup.el

```diff
@@ -141,8 +161,9 @@ that you have read this message.")
 
     ;; not relevant for stable releases
     (unless (or
-             (not (s-contains? "SNAPSHOT" ensime-server-version))
-             (--find (s-matches-p "ensime.*SNAPSHOT" it) ensime-server-jars))
+             (is-dev-version? ensime-server-version)
+             (is-server-jars-has-dev-version? ensime-server-jars)
+             )
       (error (concat
```

Comment:
I think we can skip this check. I updated the sbt plugin to download the unstable version by default (since stable ensime-emacs uses its own server installer)